### ISSUE TITLE
Announce command completely broken

### ISSUE THREAD

**Issue by j-dogcoder** [Heptagram-Bot/Heptagram#294]

**Comment by Xebec19** [comment]
.take

**Comment by monzilnepali** [comment]
can I take this one?

**Comment by j-dogcoder** [comment]
Sure!

**Comment by anubhav06** [comment]
Hi, this command just has to send the specified message, in a specified channel , right ?

If no one is working on this, Can I do this ?

**Comment by j-dogcoder** [comment]
> Hi, this command just has to send the specified message, in a specified channel , right ?
> 
> If no one is working on this, Can I do this ?

That is correct @anubhav06. I belive @monzilnepali is working on this though.

**Comment by anubhav06** [comment]
@monzilnepali Pls reply if you are still working on this issue.

**Comment by monzilnepali** [comment]
@anubhav06 Yes, I started looking into this issue but got busy. I'm planning to start working on this as soon as today, but if you're interested, you can take this one as well.

**Comment by anubhav06** [comment]
Naah, its ok if you are doing. I thought that you had become inactive like someone above who first took this issue and then unassigned himself
